src/AutorestSwift/JsonRpc/AutorestPlugin.swift

```diff
@@ -140,12 +138,19 @@ class AutorestPlugin {
             }
         }
         future.whenFailure { error in
-             FileLogger.shared.logAndFail("Call ReadFile failure \(error.localizedDescription)")
+            FileLogger.shared.logAndFail("Call ReadFile failure \(error.localizedDescription)")
         }
     }
 
     func handleReadFile(response: RPCObject) {
-        let codeModel = response.asString ?? ""
-        // TODO: Now convert to code model and generate
+        guard let codeModel = response.asString else {
+            FileLogger.shared.logAndFail("Unable to retrieve code model from Autorest.")
+        }
+        let manager = Manager(withString: codeModel)
+        do {
+            try manager.run()
+        } catch {
+            FileLogger.shared.logAndFail("Code generation failure: \(error)")
```

Comment:
Currently this fails for me because of Stencil files.

Reply:
It works on my machine.